README.md

```diff
@@ -40,13 +40,29 @@ Money.new(1000, "USD") != Money.new(1000, "EUR")   #=> true
 # Arithmetic
 Money.new(1000, "USD") + Money.new(500, "USD") == Money.new(1500, "USD")
 Money.new(1000, "USD") - Money.new(200, "USD") == Money.new(800, "USD")
-Money.new(1000, "USD") / 5                     == Money.new(200, "USD")
 Money.new(1000, "USD") * 5                     == Money.new(5000, "USD")
 
+# Splitting money evenly
+Money.new(1000, "USD").split(2)            == [Money.new(500, "USD"), Money.new(500, "USD")]
+Money.new(1000, "USD").split(3)            == [Money.new(333.34, "USD"), Money.new(333.33, "USD"), Money.new(333.33, "USD")]
+Money.new(1000, "USD").calculate_splits(2) == { Money.new(500, "USD") => 2 }
+Money.new(1000, "USD").calculate_splits(3) == { Money.new(333.34, "USD") => 1, Money.new(333.33, "USD") =>2 }
+
+# Allocating money proportionally
+Money.new(1000, "USD").allocate([0.50, 0.25, 0.25])               == [Money.new(500, "USD"), Money.new(250, "USD"), Money.new(250, "USD")]
```

Comment:
Minor nitpick:

Thanks for expanding the docs but could you tweak the formatting a little to keep it on one page? The longer examples are being cut and adding an horizontal scrollbar 😅 

<img width="1071" alt="Screenshot 2023-08-08 at 10 08 50 AM" src="https://github.com/Shopify/money/assets/4110905/e5b63917-366e-43b9-90ab-8b4b75ee0268">